Added Relative Framenumber showing on each Timeline item

ahgsql opened the pull request:
Hi, this is my first pr, When editing components, i need to see frame number relative to current sequence and with small research, be able to do that. So All Timeline Sequences will show their frame number based on their "from" prop, if timeline cursor aboves it.

JonnyBurger commented:
Hi @ahgsql! Awesome PR, thanks a lot! I intend to merge it, it's a nice feature!

If you have time, can you restore all the package-lock.json files to the original ones? They shouldn't change. Probably it's because we used npm 7 to generate them and you used npm 6 to install it, it's a bit annoying...

JonnyBurger commented:
I did it now and made some more changes - I moved it to the left side and am rendering the time in human format, e.g. `0:00:01.34`, plus added a title label for explanation. Also reverted the timeline tracks style - it was unclear why it was changed in this PR.

Thanks a lot for the idea and the PR! 🙌

ahgsql commented:
> I did it now and made some more changes - I moved it to the left side and am rendering the time in human format, e.g. `0:00:01.34`, plus added a title label for explanation. Also reverted the timeline tracks style - it was unclear why it was changed in this PR.
> 
> Thanks a lot for the idea and the PR! 🙌

sorry for lockfiles and gradient changes, i forgot it to revert, was just experimenting. And moving to left pane was good idea
